docs/getting-started.md

```diff
@@ -74,6 +77,9 @@ On MacOS/Linux, run:
 
 ```bash
 serverless upgrade
+
+# You can also specify a major version:
+serverless upgrade --major=2
```

Comment:
Did you test this command? I believe by default `serverless upgrade` upgrades to the latest minor version and by passing `--major` flag you can indicate that you want to upgrade to latest major version.

Reply:
I'm sorry, I double-checked again and it seems there's no support to upgrade to the latest minor as it will error out when you try to upgrade saying you have to add `--major` flag to upgrade to new major. I think we have a small hole in upgrade path for standalone users if I'm not missing something. @medikoo is there a way to upgrade standalone to latest minor other than using script-based installation?

Reply:
Yes, `--major` here, is to indicate that we're fine upgrading to the new _major_ release, that's it, and it's a boolean.

Reply:
Updated to `curl -o- -L https://slss.io/install | VERSION=2.72.2 bash`